src/basic/selinux-util.c

```diff
@@ -66,32 +69,34 @@ bool mac_selinux_use(void) {
 #endif
 }
 
+bool mac_selinux_enforcing(void) {
+#if HAVE_SELINUX
+        return enforcing_status_func() != 0;
+#else
+        return false;
+#endif
+}
+
 void mac_selinux_retest(void) {
 #if HAVE_SELINUX
         cached_use = -1;
 #endif
 }
 
-int mac_selinux_init(void) {
 #if HAVE_SELINUX
+static int open_label_db(struct selabel_handle **hnd) {
```

Comment:
We typically prefix return parameters with `ret_`, see CODING_STYLE. i.e. call this `ret_handle` or so

Reply:
refactored `open_label_db()`